src/Kusto/docs/Get-AzKustoOperationsResultsLocation.md

```diff
@@ -0,0 +1,185 @@
+---
+external help file:
+Module Name: Az.Kusto
+online version: https://docs.microsoft.com/powershell/module/az.kusto/get-azkustooperationsresultslocation
+schema: 2.0.0
+---
+
+# Get-AzKustoOperationsResultsLocation
+
+## SYNOPSIS
+Returns operation results.
+
+## SYNTAX
+
+### Get (Default)
+```
+Get-AzKustoOperationsResultsLocation -Location <String> -OperationId <String> [-SubscriptionId <String[]>]
+ [-DefaultProfile <PSObject>] [-PassThru] [<CommonParameters>]
```

Comment:
There seems no particular need for `-PassThru` parameter. Use [hide directive] to hide it.
Also [rename the cmdlet](https://github.com/Azure/autorest.powershell/blob/main/docs/directives.md#cmdlet-rename) from `Get-AzKustoOperationsResultsLocation` to `Get-AzKustoOperationsResultLocation` so it's consistent with `Get-AzKustoOperationsResult`.